packaging/install-telepresence.ps1

```diff
@@ -0,0 +1,5 @@
+$current_directory = (pwd).path
+echo $current_directory
+Start-Process msiexec -Wait -verb runAs -Args "/i $current_directory\winfsp.msi /passive /qn /L*V winfsp-install.log"
+Start-Process msiexec -Wait -verb runAs -Args "/i $current_directory\sshfs-win.msi /passive /qn /L*V sshfs-win-install.log"
+[Environment]::SetEnvironmentVariable("Path", "C:\\;C:\\Program Files\\SSHFS-Win\\bin;$ENV:Path", "Machine")

```

Comment:
I mostly just added this bc it was in the circle config, but wanted to check with Jose if he thinks it's necessary all the time.

Additionally, I figured I shouldn't add Telepresence to the PATH in the ps1 and just have them do it themselves, but if we think we should automatically add it to the path, I can add that here.

Reply:
So I don't think we should add `C:\\` to the PATH here, that's there in Circle so that kubectl and other tools can be found. sshfs-win we definitely need, and the installer doesn't add it to the Path.

Other than that, I think if we're manipulating the Path we might as well add the Telepresence path, might make it a little easier for the user 🤷

Reply:
Yeah that sounds good, after sleeping on it, I also decided it made sense to include it. I'll add a comment here, and also add a comment in the docs that we are manipulating the PATH just so users know.